src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,46 @@
+@WishList
+Feature: Wish List;
+
+  Background:
+    Given user is on http://automationpractice.com/ page
+
+
+  @CreateWishList
+  Scenario Outline: Create Wish List
+    Given user is logged in
+    And user is on MY WISHLISTS page
+    And user type on name field "<name>"
+    When user click on Save button
+    Then Wishlist is created
+
+    Examples:
+      | name         |
+      | My favourite |
+      | 1235         |
+
+
+  @AddToWishList
+  Scenario: Add to wishlist/see the item
+    Given user is logged in
+    And user on main page
+    And user pointed mouse on item
+    And user click on More button
+    And user is redirected on item page
```

Comment:
You can replace this with one single step like: **user clicks on X item**

Reply:
done